### ISSUE TITLE
ImportError: cannot import name 'Sequence' from 'keras.utils' (/usr/local/lib/python3.7/dist-packages/keras/utils/__init__.py)

### ISSUE THREAD

**Issue by YueWangpl** [metal3d/keras-video-generators#46]
**System Information**
- your operating system: Colab
- your python version (`python -version`): 3.7.10
- keras video generator version (`pip freeze | grep keras-video-generators`): 1.0.14

**Describe the bug**
A clear and concise description of what the bug is. Please provide the the error output with the traceback and if possible a little code snippet to reproduce the error)

> /usr/local/lib/python3.7/dist-packages/keras_video/generator.py in <module>()
     16 log = logging.getLogger()
     17 
---> 18 from keras.utils import Sequence
     19 from keras.preprocessing.image import \
     20     ImageDataGenerator, img_to_array
ImportError: cannot import name 'Sequence' from 'keras.utils' (/usr/local/lib/python3.7/dist-packages/keras/utils/__init__.py)

Error when importing Sequence. This must be something new of Colab/ keras, or an inconsistency between tensorflow.keras and keras. Since `from tensorflow.keras.utils import Sequence` would not throw this error.

![image](https://user-images.githubusercontent.com/70791150/120949462-977fb580-c712-11eb-9676-df533ab27364.png)

**Comment by Atzon** [comment]
It seems that keras-video-generators do not support tensorflow> 2.x, google colab always has the latest stable version of tensorflow installed. You need to specify the desired tensroflow version on your notebook, i.e.
`%tensorflow_version 1.x`

and then: 
`import tensorflow
print(tensorflow.__version__)`

**Comment by metal3d** [comment]
Yes. I need to find time to fix this package but I'm very busy for months.
Really sorry, I will try to find some times today and this week to fix
everything.

**Comment by metal3d** [comment]
That's weird, I use it with TF v2... 
I think that the only one problem is that I try to use "keras" if I found it, either tensorflow.keras if not. And it could be the problem. I will take a look

**Comment by venujk** [comment]
can we solve it by cloning the repository and changing the import statement to `from tensorflow.keras.utils import Sequence`

**Comment by DR90BABY** [comment]
This worked for me using the Google collab. After printing out the tensorflow version and seein git is 2.5 I opened `/usr/local/lib/python3.7/dist-packages/keras_video/generator.py`

I replaced line 18 `from keras.utils import Sequence` with `from tensorflow.keras.utils import Sequence
`  saved it and it now works for me
[Used this Stackoverflow as a guide](https://stackoverflow.com/questions/67555954/cannot-import-name-sequence-from-keras-utils)

**Comment by metal3d** [comment]
I will fix it, sorry for the very high delay, I had too many things ...

**Comment by metal3d** [comment]
It's now theorically OK for Tensorflow 2.5. I will package a new version as soon as I finish to cleanup the source code to respect better formating, less complexity and so on.

**Comment by ZumelzuR** [comment]
I saw in the readme the new version v1.1.0, but no tags added.

**Comment by metal3d** [comment]
I will add the new tag and make a release today.
I'm very sorry for a long time without any answer and no evolution. This year was hard.

**Comment by ZhuoQian111** [comment]
I have read your essay. you are a lovely uncle i think haha

**Comment by Ankitjaiswal1201** [comment]
> This worked for me using the Google collab. After printing out the tensorflow version and seein git is 2.5 I opened `/usr/local/lib/python3.7/dist-packages/keras_video/generator.py`
> 
> I replaced line 18 `from keras.utils import Sequence` with `from tensorflow.keras.utils import Sequence ` saved it and it now works for me [Used this Stackoverflow as a guide](https://stackoverflow.com/questions/67555954/cannot-import-name-sequence-from-keras-utils)

I was still getting this error even after the lines were changed. I copied the keras_videos folder in my working directory and this error was resolved.

**Comment by ghost** [comment]
ImportError: cannot import name 'Sequence' from 'keras.utils' (/usr/local/lib/python3.7/dist-packages/keras/utils/__init__.py)

ImportError: cannot import name 'Sequence' from 'keras.utils' (/usr/local/lib/python3.7/dist-packages/keras/utils/__init__.py)

ImportError: cannot import name 'Sequence' from 'keras.utils' (/usr/local/lib/python3.7/dist-packages/keras/utils/__init__.py)

**Comment by ghost** [comment]
> I will add the new tag and make a release today. I'm very sorry for a long time without any answer and no evolution. This year was hard.

ImportError: cannot import name 'Sequence' from 'keras.utils' (/usr/local/lib/python3.7/dist-packages/keras/utils/__init__.py)

**Comment by rizki4106** [comment]
Hi, i've read to all the code and i think there's no problem with the code but `the problem is` the package on pypi site haven't update yet with the newst code. here is one of the solution to fix that :

* try to clone this repo 
* change the `version number` in `setup.py` 
* then try to run 
```bash
python3 setup.py sdist
```
```bash
python3 setup.py install
```
after that try to import it again, it will works.

or you just need to download source code from here [https://github.com/metal3d/keras-video-generators/raw/master/src/keras_video/generator.py](https://github.com/metal3d/keras-video-generators/raw/master/src/keras_video/generator.py) if you want just `VideoFrameGenerator`

and then use it like below, it's works as well

```python
from generator import VideoFrameGenerator
```

hi sir @metal3d please update the package on pypi with the newest code base

**Comment by adolfojara10** [comment]
> > This worked for me using the Google collab. After printing out the tensorflow version and seein git is 2.5 I opened `/usr/local/lib/python3.7/dist-packages/keras_video/generator.py`
> > I replaced line 18 `from keras.utils import Sequence` with `from tensorflow.keras.utils import Sequence ` saved it and it now works for me [Used this Stackoverflow as a guide](https://stackoverflow.com/questions/67555954/cannot-import-name-sequence-from-keras-utils)
> 
> I was still getting this error even after the lines were changed. I copied the keras_videos folder in my working directory and this error was resolved.

Thank you!!! It worked for me
